### PR TITLE
Removes Bay's console news stories reader/browser and all stories

### LINKED ISSUE #534: SPACE magazine on the computers
#### Description of issue

It should probably be removed relatively quickly given it goes into lots of detail about BayLore and Nyx and such.

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/Baystation12/Baystation12/issues)

### PR DESCRIPTION
**Pull request by EmperorJon**
I'm assuming I don't need a changelog for purging something that will "never" have existed on this codebase.

(Fixes #534)

### CODE REVIEW

File: code/modules/modular_computers/file_system/news_article.dm [hunk 1]
```diff
@@ -14,20 +14,4 @@
...
-
-/datum/computer_file/data/news_article/space/vol_three
-	filename = "SPACE Magazine vol. 3"
-	server_file_path = 'news_articles/space_magazine_3.html'
-
-/datum/computer_file/data/news_article/space/vol_four
-	filename = "SPACE Magazine vol. 4"
-	server_file_path = 'news_articles/space_magazine_4.html'
\ No newline at end of file
+// NEWS DEFINITIONS BELOW THIS LINE
```
Comment (Techhead0) [inline code/modules/modular_computers/file_system/news_article.dm]: Kill this line too.
Unless we're gonna have new-news?

Comment (EmperorJon) [inline code/modules/modular_computers/file_system/news_article.dm]: No idea. If there's going to be no news ever you could just remove the entire program. I was just cleaning up the Bay-specific stuff that's there so people don't get confused.

**Comment by EmperorJon** [comment]
Removes the newsbrowser entirely.

**Comment by Zuhayr** [comment]
:+1:

**Pull request merged**